app/views/results/edit.html.erb

```diff
@@ -134,6 +135,39 @@
                        can_download: true } %>
 
 </aside>
+<aside class='dialog' id='help_dialog'>
+  <h1>Keyboard Shortcut Help</h1>
+  <table class="help_table">
+    <tr>
+      <td>Shift + ></td>
```

Comment:
Please use [`kbd`](https://developer.mozilla.org/en/docs/Web/HTML/Element/kbd) HTML elements for these. You can add some CSS to style them if you want.